<?xml version="1.0" encoding="UTF-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="de">
 <head>
   <title>MGX metagenome analysis platform</title>
   <link rel="stylesheet" media="all" href="main.css" />
 </head>
 <body>
<br>
   <img class="logo" src="mgx_logo.png"></img>
   <img class="logo_right" src="denbi_logo.png"></img>
   <h1>MGX metagenome analysis platform</h1>

   <div>
<p>
The characterization of microbial communities based on
sequencing and analysis of their genetic information has become a popular
approach also referred to as metagenomics; in particular, the recent advances in
sequencing technologies have enabled researchers to study even the most
complex communities.
Metagenome analysis, the assignment of sequences to taxonomic and
functional entities, however, remains a tedious task: large amounts of data need
to be processed. There are a number of approaches addressing particular aspects,
but scientific questions are often too specific to be answered by a general-purpose
method.
We present MGX, a flexible and extensible client/server-framework for
the management and analysis of metagenomic datasets; MGX features a
comprehensive set of adaptable workflows required for taxonomic and functional
metagenome analysis, combined with an intuitive and easy-to-use graphical user
interface offering customizable result visualizations. At the same time, MGX
allows to include own data sources and devise custom analysis pipelines, thus
enabling researchers to perform basic as well as highly specific analyses within a
single application.
With MGX, we provide a novel metagenome analysis platform
giving researchers access to the most recent analysis tools. MGX covers
taxonomic and functional metagenome analysis, statistical evaluation and a wide
range of visualizations easing data interpretation. Its default taxonomic
classification pipeline provides equivalent or superior results in comparison to
existing tools.
</p>
   </div>

<div class="slideshow-container" >

  <!-- Full-width images with number and caption text -->
  <div class="mySlides fade">
    <img src="slides/1.png" style="width:80%">
  </div>

  <div class="mySlides fade">
    <img src="slides/2.png" style="width:80%">
  </div>

  <div class="mySlides fade">
    <img src="slides/3.png" style="width:80%">
  </div>

  <div class="mySlides fade">
    <img src="slides/4.png" style="width:80%">
  </div>

  <div class="mySlides fade">
    <img src="slides/5.png" style="width:80%">
  </div>

  <div class="mySlides fade">
    <img src="slides/6.png" style="width:80%">
  </div>

  <div class="mySlides fade">
    <img src="slides/7.png" style="width:80%">
  </div>

  <div class="mySlides fade">
    <img src="slides/8.png" style="width:80%">
  </div>

  <div class="mySlides fade">
    <img src="slides/9.png" style="width:80%">
  </div>

  <div class="mySlides fade">
    <img src="slides/10.png" style="width:80%">
  </div>

  <!-- Next and previous buttons -->
  <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
  <a class="next" onclick="plusSlides(1)">&#10095;</a>
</div>
<br>
<!-- The dots/circles -->
<div style="text-align:center">
  <span class="dot" onclick="currentSlide(1)"></span>
  <span class="dot" onclick="currentSlide(2)"></span>
  <span class="dot" onclick="currentSlide(3)"></span>
  <span class="dot" onclick="currentSlide(4)"></span>
  <span class="dot" onclick="currentSlide(5)"></span>
  <span class="dot" onclick="currentSlide(6)"></span>
  <span class="dot" onclick="currentSlide(7)"></span>
  <span class="dot" onclick="currentSlide(8)"></span>
  <span class="dot" onclick="currentSlide(9)"></span>
  <span class="dot" onclick="currentSlide(10)"></span>
</div>

<script>
 var slideIndex = 1;
showSlides(slideIndex);

// Next/previous controls
function plusSlides(n) {
  showSlides(slideIndex += n);
}

// Thumbnail image controls
function currentSlide(n) {
  showSlides(slideIndex = n);
}

function showSlides(n) {
  var i;
  var slides = document.getElementsByClassName("mySlides");
  var dots = document.getElementsByClassName("dot");
  if (n > slides.length) {slideIndex = 1}
  if (n < 1) {slideIndex = slides.length}
  for (i = 0; i < slides.length; i++) {
      slides[i].style.display = "none";
  }
  for (i = 0; i < dots.length; i++) {
      dots[i].className = dots[i].className.replace(" active", "");
  }
  slides[slideIndex-1].style.display = "block";
  dots[slideIndex-1].className += " active";
}
</script>

  <h3>Features</h2>
  <p>
    <ul>
      <li>Data and metadata capture</li>
      <li>Quality control reports</li>
      <li>analysis <a href="https://github.com/MGX-metagenomics/pipelines">pipelines</a> for taxonomic and functional analysis of raw reads</li>
      <li>metagenome (co-)assembly and taxonomic binning for MAG recovery</li>
      <li>de novo metatranscriptome assembly</li>
      <li>workflows for taxonomic and functional annotation of assemblies</li>
      <li>High-quality charts and interactive visualizations</li>
      <li>Statistics: PCA, PCoA, Rarefaction, ..</li>
      <li>Fragment recruitment</li>
      <li>Sequence export</li>
      <li>Custom pipeline support</li>
    </ul>
  </p>

  <h3>News</h3>
  <p>
    <ul>
      <li>2023-09-21: Public release of MGX 2.0, which supports metagenome coassembly, taxonomic binning and de novo metatranscriptome assembly</li>
      <li>2020-12-23: The <a href="https://www.cebitec.uni-bielefeld.de">Center for Biotechnology</a> has discontinued its FTP service; benchmark datasets used in the <a href="https://doi.org/10.1186/s40168-018-0460-1">MGX manuscript</a> have been moved to <a href="https://zenodo.org">Zenodo</a> and can be retrieved using the <a href="https://doi.org/10.5281/zenodo.4382271">10.5281/zenodo.4382271</a> DOI.</li>
      <li>2020-04-06: With the most recent database update, the default taxonomic classification workflow employed
  by MGX is now also able to recognize and classify <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a> sequences.</li>
    </ul>
  </p>

  <h3>MGX software installation</h3>
  <p>
    The latest version of the MGX application is provided as a platform-independent .zip archive from the <a href="https://github.com/MGX-metagenomics/MGX-gui/releases">releases</a> page. No installation is required, just extract the .zip archive.
    If you are a previous user of MGX 1.0, make sure to download the latest MGX-(date).zip file; for MGX 2.0, all
    releases are prefixed with MGX2 followed by a timestamp.
  </p>

  <h3>Documentation</h2>
  <p>
    The MGX user guide is available as an online 
    <a href="guide/index.html">HTML</a> or
    <a href="https://github.com/MGX-metagenomics/userguide/releases">PDF</a>
    version. MGX is academic research software; no warranty is expressed or implied.
  </p>

  <h3>Public demonstration projects</h3>
  <p>
    If you want to try out MGX 2.0, we offer two public demonstration projects:
    <ul>
      <li>MGX2_Demo contains sequence data from the black band disease study previously
          <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5240343/">published</a> by
          Sato <i>et al.</i> that was used as a demonstration example in the upcoming MGX 2.0 publication 
          (accession <a href="https://www.ebi.ac.uk/ena/browser/view/PRJNA321354">PRJNA321354</a>)
      </li>
      <li>MGX2_TermiteGut with metagenome and metatranscriptome date from the well-known termite gut
          study; ENA project accession <a href="https://www.ebi.ac.uk/ena/browser/view/PRJNA454696">PRJNA454696</a>.
      </li>
    </ul>
    <p>
    To access these example projects, download the MGX 2.0 application and log in with username "mgxdemo"
    and password "mgxdemo". Please note that only read access is permitted here, <i>i.e.</i> you can explore
    all precomputed results and inspect the assemblies, but wont be able to initiate new analyses or 
    delete existing data.
    </p>
  </p>

  <h3>Private project request</h3>
  <p>
    We operate and maintain <s>two</s> one public MGX server instances hosted at the <a href="https://www.computational.bio.uni-giessen.de/">Bioinformatics and Systems Biology</a> group at <a href="https://www.uni-giessen.de/">JLU Giessen</a>
    <s>and <a href="https://www.cebitec.uni-bielefeld.de">Center for Biotechnology</a>, <a href="https://www.uni-bielefeld.de">Bielefeld University</a></s>. To
    obtain a protected project for analysis of own metagenome data, please 
    <a href="mailto:mgx@computational.bio.uni-giessen.de?subject=MGX%20private%20project%20request">get in touch with us</a> and provide the following
    information:
    
    <ul>
      <li>suggested short project name, e.g. MGX_AcidMine</li>
      <li>a one-line description of your project, e.g. Acid Mine Drainage metagenome</li>
      <li>a contact address of a single person responsible for the project (PI or group leader)</li>
    </ul>
  </p>
  <p>
  All data submitted to MGX is kept confidential; we encourage submission of metadata and sequence datasets to the public <a href="https://www.insdc.org/">INSDC</a> databases.
  </p>

  <h3>Source code availability</h3>
  <p>
    The MGX source code for the client library, GUI and all required server components is publicly available from the GitHub <a href="https://github.com/MGX-metagenomics/">repository</a>.
  </p>

  <h3>Citation</h3>
  <p>
    <i>MGX 2.0: Shotgun- and assembly-based metagenome and metatranscriptome analysis from a single source</i><br/>
    Sebastian Jaenicke, Sonja Diedrich, and Alexander Goesmann<br/>
    bioRxiv 2023.09.21.558800<br/>
    DOI: <a href="https://www.biorxiv.org/content/10.1101/2023.09.21.558800v1">10.1101/2023.09.21.558800</a><br/><br/>
  </p>
  <p>
    <i>Flexible metagenome analysis using the MGX framework</i><br/>
    Sebastian Jaenicke, Stefan P. Albaum, Patrick Blumenkamp, Burkhard Linke, Jens Stoye and Alexander Goesmann<br/>
    Microbiome 2018 <b>6</b>:76<br/>
    DOI: <a href="https://doi.org/10.1186/s40168-018-0460-1">10.1186/s40168-018-0460-1</a><br/><br/>

    Export citation in <a href="mgx.bib">BibTeX</a> or <a href="mgx.enw">EndNote</a> format.
  </p>

  <h3>Funding</h3>
  <img class="logo_small" src="denbi_logo.png" width="15%"></img>
  <p>
    Funding for the operation and maintenance of MGX is provided by the 
    <a href="https://www.bmbf.de">German Federal Ministry of Education and Research (BMBF)</a> project 
    <q><a href="https://www.denbi.de/bigi">Bielefeld-Gie&szlig;en Center for Microbial Bioinformatics - BiGi</a></q> (grant 031A533) within the
    German Network for Bioinformatics Infrastructure (<a href="https://www.denbi.de/">de.NBI</a>).
  </p>

  <h3>Additional links</h3>
  <p>
    <ul>
      <li>Synthetic metagenomes used in the MGX manuscript are available from <del>the <a href="https://www.cebitec.uni-bielefeld.de/">CeBiTec</a> FTP <a href="ftp://ftp.cebitec.uni-bielefeld.de/pub/software/mgx/benchmark_data/">server</a></del> <a href="https://doi.org/10.5281/zenodo.4382271">Zenodo</a></li>
      <li><a href="https://www.computational.bio.uni-giessen.de/">Bioinformatics and Systems Biology at JLU Giessen</a></li>
      <li><a href="https://www.uni-giessen.de/fbz/fb08/Inst/bioinformatik/bcf/data-privacy">Data privacy statement</a></li>
      <li><a href="https://www.denbi.de/">de.NBI - German network for bioinformatics infrastructure</a></li>
      <li><a href="https://www.kegg.jp/feedback/copyright.html">KEGG copyright request</a> (needed if you want to publish a KEGG pathway map created within MGX)</li>
    </ul>
  </p>

 <hr>
 <p class="footer">
 <a href="https://www.uni-giessen.de/de/fbz/fb08/Inst/bioinformatik/people/current_members/sebastianjaenicke">Sebastian Jaenicke</a>, 2023-09-25
 </p>

 </body>
</html>
