<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<!--Converted with LaTeX2HTML 2018 (Released Feb 1, 2018) -->
<HTML>
<HEAD>
<TITLE>Importing reference genomes</TITLE>
<META NAME="description" CONTENT="Importing reference genomes">
<META NAME="keywords" CONTENT="main">
<META NAME="resource-type" CONTENT="document">
<META NAME="distribution" CONTENT="global">

<META NAME="Generator" CONTENT="LaTeX2HTML v2018">
<META HTTP-EQUIV="Content-Style-Type" CONTENT="text/css">

<LINK REL="STYLESHEET" HREF="main.css">

<LINK REL="next" HREF="node28.html">
<LINK REL="previous" HREF="node26.html">
<LINK REL="up" HREF="node26.html">
<LINK REL="next" HREF="node28.html">
</HEAD>

<BODY >

<DIV CLASS="navigation"><!--Navigation Panel-->
<A NAME="tex2html478"
  HREF="node28.html">
<IMG WIDTH="37" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="next" SRC="next.png"></A> 
<A NAME="tex2html474"
  HREF="node26.html">
<IMG WIDTH="26" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="up" SRC="up.png"></A> 
<A NAME="tex2html468"
  HREF="node26.html">
<IMG WIDTH="63" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="previous" SRC="prev.png"></A> 
<A NAME="tex2html476"
  HREF="node1.html">
<IMG WIDTH="65" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="contents" SRC="contents.png"></A>  
<BR>
<B> Next:</B> <A NAME="tex2html479"
  HREF="node28.html">Displaying reference mappings</A>
<B> Up:</B> <A NAME="tex2html475"
  HREF="node26.html">Reference mapping</A>
<B> Previous:</B> <A NAME="tex2html469"
  HREF="node26.html">Reference mapping</A>
 &nbsp; <B>  <A NAME="tex2html477"
  HREF="node1.html">Contents</A></B> 
<BR>
<BR></DIV>
<!--End of Navigation Panel-->

<H2><A NAME="SECTION003111000000000000000">
Importing reference genomes</A>
</H2>

<P>
MGX provides several analysis pipelines to align metagenome reads to reference genomes;
before these pipelines can be used, the corresponding reference genome has to be added
to the project. There are two possible ways to achieve this: The MGX repository hosts
published and annotated reference genomes obtained from the NCBI; in addition, users
may choose to upload a custom reference sequence in FASTA, GenBank or EMBL format, e.g.
a finished but unpublished genome not available from official sources.
<BR>
<P>

<DIV ALIGN="CENTER"><A NAME="addref"></A><A NAME="375"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure 2.31:</STRONG>
Reference sequences to be used as mapping targets may be
imported from the global repository or uploaded in EMBL/GenBank/FASTA format.</CAPTION>
<TR><TD>
<DIV ALIGN="CENTER">

</DIV><IMG
  WIDTH="457" HEIGHT="115" ALIGN="BOTTOM" BORDER="0"
 SRC="./addReference.png"
 ALT="Image addReference"></TD></TR>
</TABLE>
</DIV>

<P>
To add a reference genome, right-click on the &ldquo;Reference sequences&rdquo; node within the
project view and select either &ldquo;Add reference&rdquo; to access the MGX repository or
&ldquo;Upload EMBL/GenBank/FASTA reference&rdquo; to provide an own sequence (<A HREF="#addref">2.31</A>).
<BR>
<P>
Once the import is complete, the reference genome is available for analysis and
can be selected for the corresponding analysis pipelines which provide reference
mapping, e.g. BowTie or FR-HIT.

<P>
<BR><HR>
<ADDRESS>
Sebastian Jaenicke, 2020-04-28
</ADDRESS>
</BODY>
</HTML>
