<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<!--Converted with LaTeX2HTML 2018 (Released Feb 1, 2018) -->
<HTML>
<HEAD>
<TITLE>Assembling a metagenome or metatranscriptome</TITLE>
<META NAME="description" CONTENT="Assembling a metagenome or metatranscriptome">
<META NAME="keywords" CONTENT="main">
<META NAME="resource-type" CONTENT="document">
<META NAME="distribution" CONTENT="global">

<META NAME="Generator" CONTENT="LaTeX2HTML v2018">
<META HTTP-EQUIV="Content-Style-Type" CONTENT="text/css">

<LINK REL="STYLESHEET" HREF="main.css">

<LINK REL="next" HREF="node23.html">
<LINK REL="previous" HREF="node19.html">
<LINK REL="up" HREF="node9.html">
<LINK REL="next" HREF="node23.html">
</HEAD>

<BODY >

<DIV CLASS="navigation"><!--Navigation Panel-->
<A NAME="tex2html417"
  HREF="node23.html">
<IMG WIDTH="37" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="next" SRC="next.png"></A> 
<A NAME="tex2html413"
  HREF="node9.html">
<IMG WIDTH="26" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="up" SRC="up.png"></A> 
<A NAME="tex2html407"
  HREF="node21.html">
<IMG WIDTH="63" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="previous" SRC="prev.png"></A> 
<A NAME="tex2html415"
  HREF="node1.html">
<IMG WIDTH="65" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="contents" SRC="contents.png"></A>  
<BR>
<B> Next:</B> <A NAME="tex2html418"
  HREF="node23.html">Visualization of results</A>
<B> Up:</B> <A NAME="tex2html414"
  HREF="node9.html">Using the MGX application</A>
<B> Previous:</B> <A NAME="tex2html408"
  HREF="node21.html">Monitoring job progress</A>
 &nbsp; <B>  <A NAME="tex2html416"
  HREF="node1.html">Contents</A></B> 
<BR>
<BR></DIV>
<!--End of Navigation Panel-->

<H1><A NAME="SECTION00390000000000000000">
Assembling a metagenome or metatranscriptome</A>
</H1>

<P>
Since MGX 2.0, we support the assembly and analysis of both metagenomes and metatranscriptomes; for this, we
provide workflows targeting QC, (co-)assembly, gene prediction, taxonomic binning, completeness assessment, and
abundance quantification. For metatranscriptomes, an additional step is implemented for the removal of ribosomal
RNA sequences prior to the assembly, and binning is not performed.
<BR>
<P>
To create an assembly, select all sequencing runs that should be co-assembled within the Project Explorer window
and select the &ldquo;Assemble&rdquo; entry from the context menu. Single-end data can be included into this process, but
the corresponding workflows require that at least one paired-end dataset is provided, as this yields better assembly
results. Also, only paired-end data is used for the taxonomic binning step, and we recommend to co-assemble multiple 
paired-end datasets for an improved binning outcome.
<BR>
<P>
The assembly wizard requires to specify a name for the new assembly, and once the assembly workflow has been
submitted, its state can be monitored from the Job Monitor (see above).
<BR><HR>
<ADDRESS>
Sebastian Jaenicke, 2024-02-18
</ADDRESS>
</BODY>
</HTML>
