<!DOCTYPE HTML>

<!--Converted with LaTeX2HTML 2021.2 (Released July 1, 2021) -->
<HTML lang="en">
<HEAD>
<TITLE>Quality control</TITLE>
<META NAME="description" CONTENT="Quality control">
<META NAME="keywords" CONTENT="main">
<META NAME="resource-type" CONTENT="document">
<META NAME="distribution" CONTENT="global">

<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
<META NAME="viewport" CONTENT="width=device-width, initial-scale=1.0">
<META NAME="Generator" CONTENT="LaTeX2HTML v2021.2">

<LINK REL="STYLESHEET" HREF="main.css">

<LINK REL="next" HREF="node19.html">
<LINK REL="previous" HREF="node17.html">
<LINK REL="next" HREF="node19.html">
</HEAD>

<BODY >

<DIV CLASS="navigation"><!--Navigation Panel-->
<A
 HREF="node19.html">
<IMG WIDTH="37" HEIGHT="24" ALT="next" SRC="next.png"></A> 
<A
 HREF="node9.html">
<IMG WIDTH="26" HEIGHT="24" ALT="up" SRC="up.png"></A> 
<A
 HREF="node17.html">
<IMG WIDTH="63" HEIGHT="24" ALT="previous" SRC="prev.png"></A> 
<A ID="tex2html195"
  HREF="node1.html">
<IMG WIDTH="65" HEIGHT="24" ALT="contents" SRC="contents.png"></A>  
<BR>
<B> Next:</B> <A
 HREF="node19.html">Defining and executing analysis</A>
<B> Up:</B> <A
 HREF="node9.html">Using the MGX application</A>
<B> Previous:</B> <A
 HREF="node17.html">Importing sequence data</A>
 &nbsp; <B>  <A ID="tex2html196"
  HREF="node1.html">Contents</A></B> 
<BR>
<BR></DIV>
<!--End of Navigation Panel-->

<H1><A ID="SECTION00370000000000000000">
Quality control</A>
</H1>

<P>

<DIV class="CENTER"><A ID="qcopen"></A><A ID="245"></A>
<TABLE>
<CAPTION class="BOTTOM"><STRONG>Figure 2.14:</STRONG>
After selecting a sequencing run object, the Quality Control component can be opened
from its menubar icon (red circle).</CAPTION>
<TR><TD>
<DIV class="CENTER"><IMG
  WIDTH="372" HEIGHT="334" STYLE=""
 SRC="./QCopen.png"
 ALT="Image QCopen">
</DIV></TD></TR>
</TABLE>
</DIV>

<P>
After sequence import, quality control reports generated within MGX should be inspected (Fig.&nbsp;<A HREF="#qcopen">2.14</A>) before proceding with data analysis.
MGX currently offers three types of QC reports: Distribution of GC content, sequence length
and nucleotide distribution within the DNA sequences. Those can be used to evaluate overall
sequence data quality and check for possible signs of contamination.

<P>

<DIV class="CENTER"><A ID="qc1"></A><A ID="251"></A>
<TABLE>
<CAPTION class="BOTTOM"><STRONG>Figure 2.15:</STRONG>
GC distribution of a public metagenome dataset (SRR3569370).</CAPTION>
<TR><TD>
<DIV class="CENTER"><IMG
 STYLE="height: 45.88ex; vertical-align: -0.11ex; " SRC="img5.svg"
 ALT="\includegraphics[width=.6\textwidth]{img/mgx/QCgc}">
</DIV></TD></TR>
</TABLE>
</DIV>

<P>

<DIV class="CENTER"><A ID="qc2"></A><A ID="256"></A>
<TABLE>
<CAPTION class="BOTTOM"><STRONG>Figure 2.16:</STRONG>
Nucleotide distribution of the public metagenome. The chart displays the distribution
of nucleotides within the first 100 bp.</CAPTION>
<TR><TD>
<DIV class="CENTER"><IMG
 STYLE="height: 45.16ex; vertical-align: -0.11ex; " SRC="img6.svg"
 ALT="\includegraphics[width=.6\textwidth]{img/mgx/QCnuc}">
</DIV></TD></TR>
</TABLE>
</DIV>

<P>

<DIV class="CENTER"><A ID="qc3"></A><A ID="261"></A>
<TABLE>
<CAPTION class="BOTTOM"><STRONG>Figure 2.17:</STRONG>
Read length distribution of the metagenome dataset; here the sequencing
data was imported without prior quality trimming.</CAPTION>
<TR><TD>
<DIV class="CENTER"><IMG
 STYLE="height: 45.72ex; vertical-align: -0.11ex; " SRC="img7.svg"
 ALT="\includegraphics[width=.6\textwidth]{img/mgx/QCreadlen}">
</DIV></TD></TR>
</TABLE>
</DIV>

<P>

<DIV class="CENTER"><A ID="qc4"></A><A ID="280"></A>
<TABLE>
<CAPTION class="BOTTOM"><STRONG>Figure 2.18:</STRONG>
Nucleotide distribution examples.</CAPTION>
<TR><TD>
<DIV class="CENTER">        <BR>
<IMG
 STYLE="height: 26.03ex; " SRC="img8.svg"
 ALT="\begin{subfigure}
% latex2html id marker 265
[b]{0.3\textwidth}
\includegraphic...
...=\textwidth]{img/mgx/highGCnucl}
\caption{High-GC (65\%) data}
\end{subfigure}">
<BR><BR>
<IMG
 STYLE="height: 26.03ex; " SRC="img9.svg"
 ALT="\begin{subfigure}
% latex2html id marker 270
[b]{0.3\textwidth}
\includegraphics[width=\textwidth]{img/mgx/ampliconNucl}
\caption{Amplicon data}
\end{subfigure}">
<BR>
        <BR>
<IMG
 STYLE="height: 25.88ex; " SRC="img10.svg"
 ALT="\begin{subfigure}
% latex2html id marker 275
[b]{0.3\textwidth}
\includegraphic...
...dth=\textwidth]{img/mgx/adapterNucl}
\caption{Adapter residue}
\end{subfigure}"></DIV></TD></TR>
</TABLE>
</DIV>

<P>
Depending on the kind of sequence data, different patterns may emerge (Fig.&nbsp;<A HREF="#qc4">2.18</A>),
which might or might not warrant any further action. While small amounts of <SPAN  CLASS="textit">e.g.</SPAN>&nbsp;adapter
residue are sometimes encountered and might be considered acceptable, it is up to the
individual researcher to check back with their sequencing provider and ask for
adapter sequences to execute additional trimming.

<P>

<DIV CLASS="navigation"><HR>
<!--Navigation Panel-->
<A
 HREF="node19.html">
<IMG WIDTH="37" HEIGHT="24" ALT="next" SRC="next.png"></A> 
<A
 HREF="node9.html">
<IMG WIDTH="26" HEIGHT="24" ALT="up" SRC="up.png"></A> 
<A
 HREF="node17.html">
<IMG WIDTH="63" HEIGHT="24" ALT="previous" SRC="prev.png"></A> 
<A ID="tex2html195"
  HREF="node1.html">
<IMG WIDTH="65" HEIGHT="24" ALT="contents" SRC="contents.png"></A>  
<BR>
<B> Next:</B> <A
 HREF="node19.html">Defining and executing analysis</A>
<B> Up:</B> <A
 HREF="node9.html">Using the MGX application</A>
<B> Previous:</B> <A
 HREF="node17.html">Importing sequence data</A>
 &nbsp; <B>  <A ID="tex2html196"
  HREF="node1.html">Contents</A></B> </DIV>
<!--End of Navigation Panel-->
<ADDRESS>
Sebastian Jaenicke, 2024-02-18
</ADDRESS>
</BODY>
</HTML>
